ads/fakead3p.js

```diff
@@ -24,4 +24,23 @@ export function fakead3p(global, data) {
   if (data.ad_container) {
     dev().assert(data.container == data.ad_container, 'wrong container');
   }
+  if (data.valid && data.valid == 'true') {
+    const img = document.createElement('img');
+    if (data.url) {
+      img.setAttribute('src', data.url);
+    }
+    let width, height;
+    if (data.iframeheight) {
```

Comment:
let's call them `adHeight` and `adWidth` which correspond to `data-ad-height` and `data-ad-width` in HTML